src/components/Join.js

```diff
@@ -257,15 +256,15 @@ const JoinContent = compose(
                               <div>
                                 <Trans i18nKey="join.wizard.profileOrBrowse">
                                   Fill out your profile, or skip and
-                                  <ToSearchPage
-                                    index="file"
+                                  <Link
+                                    to={`/search/file`}
```

Comment:
If we're making this change here it'd make sense to me to nix `ToSearchPage` altogether & convert the other instance of it to just a `Link` as well

Reply:
oh yeah I just forgot to remove the `ToSearchPage` component, everything else is converted to just Link now :P